serendipity_event_dejure/serendipity_event_dejure.php

```diff
@@ -204,7 +230,9 @@ function djo_zwischenspeicherung() {
 				$text = $vernetzung[1];
 			}
     			serendipity_db_Query("DELETE FROM {$serendipity['dbPrefix']}dejure WHERE ckey = '" . $schluessel . "'");
-    			serendipity_db_Query("INSERT INTO {$serendipity['dbPrefix']}dejure (ckey, cache, last_update) VALUES ('" . $schluessel . "', '" . serendipity_db_escape_string($text) . "', " . time() . ")");
+                serendipity_db_Query("INSERT IGNORE INTO {$serendipity['dbPrefix']}dejure
```

Comment:
Oh, okay. That's a mistake - "IGNORE" should not be necessary as we DELETE a (possible) entry beforehand (what upstream doesn't do). I'll push a fixed version.